### ISSUE TITLE
ofxAssimpModelLoader linker errors on OSX 10.8 / master branch.

### ISSUE THREAD

**Issue by bakercp** [openframeworks/openFrameworks#4412]
Compiling the ofxAssimpModelLoader example on 10.8 with the latest master branch produces this linking error:

![screenshot_10_5_15__9_18_pm](https://cloud.githubusercontent.com/assets/300484/10298617/a2fab2f2-6ba6-11e5-8888-3725adc94f9e.png)

Trying to recompile it on 10.8 now to see if that fixes the issue.

**Comment by arturoc** [comment]
this seems like the same problem as with other libraries, can you recompile it under 10.8 as you did with the others?

**Comment by danoli3** [comment]
Yeah 
```
extern double __exp10(double) __OSX_AVAILABLE_STARTING(__MAC_10_9, __IPHONE_7_0);
```

Note: ```__MAC_10_9```

**Comment by NickHardeman** [comment]
I get this error when compiling the 09RC2 assimpExample in Xcode on OSX 10.8:
dyld: Symbol not found: ___sincos_stret
  Referenced from: /Applications/OpenFrameworks/of_v0.9.0RC2_osx_release/examples/addons/assimpExample/bin/assimpExampleDebug.app/Contents/MacOS/assimpExampleDebug
  Expected in: /usr/lib/libSystem.B.dylib

The error is fixed and the example compiles when rebuilding the Assimp lib using apothecary and specifying the 10.8 SDK. 
`cmake -G 'Unix Makefiles' -DCMAKE_OSX_DEPLOYMENT_TARGET=10.8 -DCMAKE_OSX_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.8.sdk $buildOpts -DCMAKE_C_FLAGS="-arch i386 -arch x86_64 -O3 -DNDEBUG -funroll-loops" -DCMAKE_CXX_FLAGS="-arch i386 -arch x86_64 -stdlib=libc++ -O3 -DNDEBUG -funroll-loops" .`

**Comment by arturoc** [comment]
@NickHardeman can you send a PR with the recompiled libraries?

**Comment by kylemcdonald** [comment]
closed by #4487
